Task_TheGridProblem/index.css

```diff
@@ -1 +1,7 @@
-/* Write code below for GRID PROBLEM TASK */
+/* Write code below for GRID PROBLEM TASK */

```

Comment:
The class selector '.dd' does not provide sufficient information about what this class is for, consider using a more descriptive name.